servicedirectory/api/ServiceDirectory.Samples/ServiceDirectory.Samples.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="Google.Cloud.ServiceDirectory.V1Beta1" Version="1.0.0-beta02" />

```

Comment:
Can't we use `Google.Cloud.ServiceDirectory.V1` package here? cc: @amanda-tarafa

Reply:
@vinda-goog Are the samples for the stable version of the API or for the Beta version of the API. Ideally they should be for the stable version of the API in which case you should use the package suggested by @laljikanjareeya

Reply:
Hi @laljikanjareeya and @amanda-tarafa sorry for the delay here on the response to your comments.  Appreciate your help here and have done my best to address everything.  I have also switched the Service Directory package version to V1 as we are in the process of doing this with our other samples as well.